docs/source/install/rhel7.rst

```diff
@@ -0,0 +1,387 @@
+
+
+Install StackStorm components
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+  .. code-block:: bash
+
+      sudo yum install -y st2 st2mistral
+
+
+If you are not running RabbitMQ, MongoDB or PostgreSQL on the same box, or changed defauls,
+please adjust the settings:
+
+    * RabbitMQ connection at ``/etc/st2/st2.conf`` and ``/etc/mistral/mistral.conf``
+    * MongoDB at ``/etc/st2/st2.conf``
+    * PostgreSQL at ``/etc/mistral/mistral.conf``
+
+Setup Mistral Database
+~~~~~~~~~~~~~~~~~~~~~~
+
+  .. code-block:: bash
+
+    # Create Mistral DB in PostgreSQL
+    cat << EHD | sudo -u postgres psql
+    CREATE ROLE mistral WITH CREATEDB LOGIN ENCRYPTED PASSWORD 'StackStorm';
+    CREATE DATABASE mistral OWNER mistral;
+    EHD
+
+    # Setup Mistral DB tables, etc.
+    /opt/stackstorm/mistral/bin/mistral-db-manage --config-file /etc/mistral/mistral.conf upgrade head
+    # Register mistral actions
+    /opt/stackstorm/mistral/bin/mistral-db-manage --config-file /etc/mistral/mistral.conf populate
+
+Configure SSH and SUDO
+~~~~~~~~~~~~~~~~~~~~~~
+To run local and remote shell actions, StackStorm uses a special system user (default ``stanley``).
+For remote linux actions, SSH is used. It is advised to configure identity file based SSH access on all remote hosts. We also recommend configuring SSH access to localhost for running examples and testing.
+
+* Create StackStorm system user, enable passwordless sudo, and set up ssh access to "localhost" so that SSH-based action can be tried and tested locally. You will need elevated privileges to do this.
+
+  .. code-block:: bash
+
+    # Create an SSH system user (default `stanley` user may be already created)
+    useradd stanley
+    mkdir -p /home/stanley/.ssh
+    chmod 0700 /home/stanley/.ssh
+
+    # On StackStorm host, generate ssh keys
+    ssh-keygen -f /home/stanley/.ssh/stanley_rsa -P ""
+
+    # Authorize key-base acces
+    cat /home/stanley/.ssh/stanley_rsa.pub >> /home/stanley/.ssh/authorized_keys
+    chmod 0600 /home/stanley/.ssh/authorized_keys
+    chown -R stanley:stanley /home/stanley
+
+    # Enable passwordless sudo
+    echo "stanley    ALL=(ALL)       NOPASSWD: SETENV: ALL" >> /etc/sudoers.d/st2
+    chmod 0440 /etc/sudoers.d/st2
+
+    # Make sure `Defaults requiretty` is disabled in `/etc/sudoers`
+    sed -i "s/^Defaults\s\+requiretty/# Defaults requiretty/g" /etc/sudoers
+
+* Configure SSH access and enable passwordless sudo on the remote hosts which StackStorm would control
+  over SSH. Use the public key generated in the previous step; follow instructions at :ref:`config-configure-ssh`.
+  To control Windows boxes, configure access for :doc:`Windows runners </config/windows_runners>`.
+
+* Adjust configuration in ``/etc/st2/st2.conf`` if you are using a different user or path to the key:
+
+  .. sourcecode:: ini
+
+    [system_user]
+    user = stanley
+    ssh_key_file = /home/stanley/.ssh/stanley_rsa
+
```

Comment:
Ok, let's do use sudo in front of each for full consistency. I was just lazy to learn how to sudo `>>` redirects .